app/questionnaire/questionnaire_schema.py

```diff
@@ -138,9 +142,13 @@ def placeholder_transform_section_dependencies_by_block(
         return self._placeholder_transform_section_dependencies_by_block
 
     @cached_property
-    def answer_dependencies(self) -> ImmutableDict[str, set[AnswerDependent]]:
+    def answer_dependencies(self) -> ImmutableDict[str, set[Dependent]]:
         return ImmutableDict(self._answer_dependencies_map)
 
+    @cached_property
+    def list_dependencies(self) -> ImmutableDict[str, set[Dependent]]:
```

Comment:
Do we want to add a test for this in test_questionnaire_schema as we do for some of the other dependency maps?

Reply:
Good idea 👍 have added a test for it and I moved the existing test for the when rules one and renamed it given that ive changed the method